packages/input/src/FzInput.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="w-full flex flex-col gap-8" @click="inputRef?.focus()">
+    <label :class="['text-sm', computedLabelClass]">
```

Comment:
label is not binded to input, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input